doc/howtos/rdtraining/B_acl_irrules.rst

```diff
@@ -0,0 +1,528 @@
+* All real-estate agents should be able to confirm the sale of a property they
+  can manage, but we do not want them to be able to validate or mark as paid
+  any invoice in the system.
+
+.. note::
+
+    We may actually be fine with some or most of these for a small business.
+
+    Because it's easier for users to disable unnecessary security rules than it
+    is to create them from nothing, it's better to err on the side of caution
+    and limiting access: users can relax that access if necessary or convenient.
+
+Groups
+======
+
+.. seealso::
+
+    The documentation related to this topic can be found in :ref:`the security
+    reference <reference/security>`.
+
+    :ref:`The guidelines <reference/guidelines>` document the format and
+    location of master data items.
+
+.. admonition:: **Goal**
+
+    At the end of this section,
+
+    - We can make employees *real-estate agents* or *real-estate managers*.
+    - The ``admin`` user is a real-estate manager.
+    - We have a new *real-estate agent* employee with no access to invoicing
+      or administration.
+
+It would not be practical to attach individual security rules to employees any
+time we need a change so *groups* link security rules and users. They correspond
+to roles that can be assigned to employees.
+
+For most Odoo applications [#app]_ a good baseline is to have *user* and
+*manager* (or administrator) roles: the manager can change the configuration of
+the application and oversee the entirety of its use while the user can well,
+use the application [#appuser]_.
+
+This baseline seems sufficient for us:
+
+* Real estate managers can configure the system (manage available types and
+  tags) as well as oversee every property in the pipeline.
+* Real estate agents can manage the properties under their care, or properties
+  which are not specifically under the care of any agent.
+
+In keeping with Odoo's data-driven nature, a group is no more than a record of
+the ``res.groups`` model. They are normally part of a module's :ref:`master data
+<howto/rdtraining/C_data>`, defined in one of the module's data files.
+
+As simple example `can be found here <https://github.com/odoo/odoo/blob/532c083cbbe0ee6e7a940e2bdc9c677bd56b62fa/addons/hr/security/hr_security.xml#L9-L14>`_.
+
+.. admonition:: what is the *category_id*?
+
+    ``category_id`` a *module category*, it is automatically generated from the
+    :ref:`category defined in module manifest files <howto/rdtraining/03_newapp>`.
+
+    .. todo:: the demo module needs to have the category set, to
+              *Real Estate/Brokerage*, ideally the newapp part would already do
+              that
+
+.. exercise::
+
+    Create the ``security.xml`` file in the appropriate folder and add it to the
+    ``__manifest__.py`` file.
+
+    Add a record creating a group with the id ``estate_group_user``, the name
+    "Agent" and the category ``base.module_category_real_estate_brokerage``.
```

Comment:
It's not clear here that a category 'Real estate/brokerage' (or something like that) should be added to the manifest or that a record should be created in the real estate module :
`    <record id="module_category_real_estate_brokerage" model="ir.module.category">
        <field name="name">Brokerage</field>
        <field name="description">Test</field>
    </record>`

If this is intended for the new guys to scratch their heads, then ignore my comment, but I think that some clarification on how the categories are generated from the manifest would be nice.